src/inverseRobot.js

```diff
@@ -8,19 +8,14 @@
 
 function inverseRobot(robot) {
   const newRobot = {};
-  const values = Object.values(robot);
-  const keys = Object.keys(robot);
   const uniqueValues = [];
 
-  for (const i of values) {
-    if (uniqueValues.includes(i)) {
+  for (const key in robot) {
+    if (uniqueValues.includes(robot[key])) {
       return null;
     }
-    uniqueValues.push(i);
-  }
-
-  for (let i = 0; i < keys.length; i++) {
-    newRobot[values[i]] = keys[i];
+    uniqueValues.push(robot[key]);
+    newRobot[robot[key]] = key;
```

Comment:
You can do the same without the array as you're putting the keys and values of robot object into the empty array, comparing the values, keys and only after that adding the values to the empty object.